module-infrastructure/monitoring/src/main/kotlin/site/yourevents/sentry/SentryConfig.kt

```diff
@@ -0,0 +1,22 @@
+package site.yourevents.sentry
+
+import io.sentry.Sentry
+import jakarta.annotation.PostConstruct
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class SentryConfig(
+    @Value("\${sentry.dsn}") private val sentryDsn: String,
+    @Value("\${sentry.environment}") private val environment: String,
+    @Value("\${sentry.servername}") private val serverName: String
```

Comment:
현재 컨벤션은 다음과 같이 되어 있습니다.
```suggestion
    @Value("\${sentry.dsn}") 
    private val sentryDsn: String,
    
    @Value("\${sentry.environment}") 
    private val environment: String,
    
    @Value("\${sentry.servername}") 
    private val serverName: String,
```
지금 예찬님이 작성하신 방안도 가독성이 나빠보이지는 않습니다!
어떤 컨벤션을 적용할지는 @1winhyun 님 하고도 얘기를 나눠바야 할 것 같은데, 예찬님 의견을 여쭤보고 싶습니다. 😄

Reply:
저는 이 부분은 예찬님이 선호하시는 컨벤션을 적용해주시면 될 것 같습니다!!

Reply:
해당 부분은 정완님의 의견에 따르겠습니다!

Reply:
감사합니다~!! `@Value` 어노테이션이 있는 생성자 주입은 가독성을 위해 최상단에 놓고, 개행을 띄워주는 식으로 적용하면 될 것 같습니다!